### PR TITLE
Show active accounts only by default

### PR DESCRIPTION
**Pull request by guggero**
Fixes https://github.com/lightninglabs/pool/issues/147 by adding the `--show_archived` flag (defaults to `false`) to `pool accounts list`.

### CODE REVIEW

**Review by wpaulino — APPROVED** [review]

**Comment by wpaulino** [comment]
Linter errors:
```
poolrpc/trader.pb.go:10:4: could not import google.golang.org/genproto/googleapis/api/annotations (invalid package name: "") (typecheck)
	_ "google.golang.org/genproto/googleapis/api/annotations"
	  ^
cmd/pool/account.go:1: /home/travis/gopath/pkg/mod/github.com/grpc-ecosystem/grpc-gateway@v1.14.6/runtime/fieldmask.go:10:2: could not import google.golang.org/genproto/protobuf/field_mask (invalid package name: "") (typecheck)
package main
cmd/poold/main.go:1: /home/travis/gopath/pkg/mod/github.com/grpc-ecosystem/grpc-gateway@v1.14.6/runtime/fieldmask.go:10:2: could not import google.golang.org/genproto/protobuf/field_mask (invalid package name: "") (typecheck)
package main
internal/test/keys.go:1: /home/travis/gopath/pkg/mod/github.com/grpc-ecosystem/grpc-gateway@v1.14.6/runtime/fieldmask.go:10:2: could not import google.golang.org/genproto/protobuf/field_mask (invalid package name: "") (typecheck)
package test
account/watcher/log.go:1: /home/travis/gopath/pkg/mod/google.golang.org/grpc@v1.29.1/status/status.go:34:6: could not import google.golang.org/genproto/googleapis/rpc/status (invalid package name: "") (typecheck)
package watcher
auctioneer/account_subscription.go:1: /home/travis/gopath/pkg/mod/google.golang.org/grpc@v1.29.1/status/status.go:34:6: could not import google.golang.org/genproto/googleapis/rpc/status (invalid package name: "") (typecheck)
package auctioneer
chaninfo/channel_info.go:1: /home/travis/gopath/pkg/mod/google.golang.org/grpc@v1.29.1/status/status.go:34:6: could not import google.golang.org/genproto/googleapis/rpc/status (invalid package name: "") (typecheck)
package chaninfo
```

**Comment by guggero** [comment]
Was an issue with the Travis build cache. Is resolved now.

File: poolrpc/trader.proto [hunk 1]
```diff
@@ -239,6 +239,10 @@ message QuoteAccountResponse {
 }
 
 message ListAccountsRequest {
+    /*
+    Only list accounts that are still active.
+    */
+    bool active_only = 2;
```
Comment (halseth) [inline poolrpc/trader.proto:245]: what happened to 1?

**Review by halseth — APPROVED** [review]
One q regarding the protos, otherwise LGTM 👍

File: poolrpc/trader.proto (hunk 1 above)
Comment (guggero) [reply poolrpc/trader.proto]: Argh, copy/paste strikes again. Fixed.

**Pull request merged**